test/TestERC20Symbolic.sol

```diff
@@ -0,0 +1,32 @@
+pragma solidity ^0.5.16;
+
+import "./firefly/FireflyCheck.sol";
+import "../contracts/ERC20.sol";
+
+contract TestERC20Symbolic is FireflyCheck {
+    function _testTransfer() public {
+        ERC20 token;
+
+        // token = (magic happens here)
```

Comment:
```suggestion
        address owner = this.firefly_genAddress();
        uint256 initSupply = this.firefly_genUInt256();

        ERC20 token = ERC20("gold", "GLD", owner, initSupply);
```

I think we can take advantage of the `constructor` here, to build a new token https://github.com/runtimeverification/firefly-demo/blob/master/contracts/ERC20.sol#L15 (Truffle testing example documentation: https://www.trufflesuite.com/docs/truffle/testing/writing-tests-in-solidity).

Also notice that I switched the word `newSymbolic` to `gen`, because we'll be using the same "whitebox" approach for both random testing and symbolic testing, so `gen` seems to work for both those scenarios.